src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,23 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    enum Color {
+        YELLOW,
+        GREEN,
+        BLACK,
+        BLUE,
+        PINK,
+        PURPLE,
+        RED,
+        WHITE,
+        ORANGE
+    }
+
+    private static final int COLOR_COUNT = 9;
+
     public String getRandomColor() {
-        return null;
+        return Color.values()[new Random().nextInt(COLOR_COUNT)].name();
```

Comment:
Think about which variables should be local in the method and which should be class-level. Every time when your program call method getRandomColor new object of Random will be created